glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/PingUploadWorker.kt

```diff
@@ -51,6 +52,7 @@ internal inline fun <reified W : Worker> buildWorkRequest(tag: String): OneTimeW
 class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(context, params) {
     companion object {
         internal const val PING_WORKER_TAG = "mozac_service_glean_ping_upload_worker"
+        internal const val THROTTLED_BACKOFF_MILLIS = 60_000L
```

Comment:
uber nit: I think most of the other places we refer to milliseconds in a variable name using `MS`

Reply:
True, not many places, but enough to convince me (google on the other hand uses `_MILLIS`)